### ISSUE TITLE
voice transcribed using text to speech

### ISSUE THREAD

**Issue by joshua750** [csdcorp/speech_to_text#413]
Hello this plugin support voice transcribed from existing mp3 file for example if i have existing recording file i want convert this audio into text any possible way in this plugin?

**Comment by sowens-csd** [comment]
No, this is not supported. Unfortunately none of the platforms provide this capability so the plugin can't expose it.
